adcircpy/server/slurm_config.py

```diff
@@ -118,20 +120,25 @@ def _log_filename(self, log_filename):
     @property
     def _prefix(self):
         f = f'#SBATCH -D {self._run_directory}\n' \
-            f'#SBATCH -J {self._run_name}\n' \
-            f'#SBATCH -A {self._account}\n'
+            f'#SBATCH -J {self._run_name}\n'
+
+        if self._account is not None:
```

Comment:
`account` is optional on TACC, but should not be optional on ADCIRCPy

Reply:
yeah, I set it up to be a non-optional parameter that you have to specifically pass `None` to in order for it to be ignored

Reply:
Feels hacky, I'd rather be explicit than implicit.